engine/src/physics/solver/penetration_constraint/plugin.cpp

```diff
@@ -0,0 +1,126 @@
+#include "../../../collisions/narrow_phase/plugin.hpp"
```

Comment:
After rebasing on main you can use the `collisionsTag`, which I added due to previously for the shared lib PR, so that we didn't need to access the private headers. Thus, you can remove this include then.

TL;DR physics shouldn't care about collisions internals